File: dotnet_mahitha/Day8Ado/PropDemo/PropDemo/Employ.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PropDemo
{
    class Employ
    {
        public int Empno { get; set; }
        public string Name { get; set; }
        public int Basic { get; set; }

        public override string ToString()
        {
            return "Employ No  " + Empno + " Employ Name  " + Name + " Basic  " + Basic;
        }
    }
}

File: DemoProject/src/com/java/abs/Employ.java
package com.java.abs;

public class Employ {

	int empno;
	String name;
	int basic;
	
	@Override
	public String toString() {
		return "Employ [empno=" + empno + ", name=" + name + ", basic=" + basic + "]";
	}

	public Employ(int empno, String name, int basic) {
		super();
		this.empno = empno;
		this.name = name;
		this.basic = basic;
	}
	
}
